test_world: "test"
# test_world: "test"
# the selection of test test_world
# options: "test", "diamond", "museum", "hospital"

publish_interval: 0.2
# the publish interval of this package

end_point: "end"
# options: "start", "mid", "end"